packages/react/time-series/src/chartJsAdapterDateFns.ts

```diff
@@ -0,0 +1,184 @@
+/* eslint @typescript-eslint/ban-ts-comment:0 */
+// @ts-nocheck
```

Comment:
This is just a JavaScript file, so I renamed to `.ts` and put these to avoid type-checking.